.github/ISSUE_TEMPLATE/new_tap.yml

```diff
@@ -0,0 +1,40 @@
+name: 🚰 New Singer Tap
+description: Submit a new Singer tap to Meltano Hub
+title: "[Singer Tap] <title>"
+labels: []
+assignees: []
+
+body:
+- type: markdown
+  attributes:
+    value: |
+      All Singer definitions are stored in `/_data/taps/` or `/_data/targets` and logo images are in `/assets/logos/<taps or targets>`. The minimal steps for adding a tap or target are the following:
+      Reference "How to Submit a Singer Connector to MeltanoHub" [YouTube video](https://www.youtube.com/watch?v=hKbHB3LHbEA).
+
+- type: input
+  id: description
+  attributes:
+    label: Description
+    description: General description of what the company behind the API does
+  validations:
+    required: true
+
+- type: input
+  id: url
+  attributes:
+    label: Entity URL
+    description: URL of the developer documentation or website
+  validations:
+    required: true
+
+- type: input
+  id: name
+  attributes:
+    label: Name
+    description: The unique name of the connector
+
+- type: input
+  id: singer_name
+  attributes:
+    label: Singer Name
+    description: The Singer specific name of the connector. Typically tap-<name>
```

Comment:
@edgarrmondragon I'm ok with this as a first iteration, but can we include some of the steps listed in https://github.com/meltano/hub/blob/main/.gitlab/issue_templates/Add%20New%20Singer%20Connector.md as well? 

I can possibly see that having users enter data in an issue form like this could be a little annoying if it's disconnected from the PR they'd eventually have to create as well, so I'm not sure how far we want to take the YML issue definition

Reply:
@tayloramurphy yeah, I though the same so I considered doing [something for hackday](https://github.com/orgs/meltano/discussions/388#discussioncomment-2951325) around that using something like https://jsonforms.io/docs/ to create a PR on a user's behalf.

Reply:
I like this form idea but yeah I agree with @tayloramurphy that it could end up be annoying if it doesnt help with the PR.

The goal of the previous gitlab template was to basically create an issue with a checklist of things to do. Are we changing the goal to try to allow users to build their yaml/PR using the form? I almost think we should have 2 different templates, or options. One for just creating the issue itself with the todo list when someone already has the yaml definition or plans to build it manually and another for drafting the full yaml via the form for someone whos new and needs to be walked through it because they dont quite know what content is required.

Reply:
ohh @edgarrmondragon that'd be cool! If it's possible to be primarily driven off the JSON schema that'd be amazing so there's still only a SSOT.